### ISSUE TITLE
Remove logging secrets from provisioner pod

### ISSUE THREAD

**Issue by Madhu-1** [ceph/ceph-csi#134]
Logs
`
I0117 13:01:49.310886       1 controller.go:122] GRPC request: name:"pvc-07c76cb3-1a58-11e9-92a2-525400592e74" capacity_range:<required_bytes:5368709120 > volume_capabilities:<mount:<fs_type:"ext4" > access_mode:<mode:MULTI_NODE_MULTI_WRITER > > parameters:<key:"csiNodeStageSecretName" value:"csi-cephfs-secret" > parameters:<key:"csiNodeStageSecretNamespace" value:"default" > parameters:<key:"csiProvisionerSecretName" value:"csi-cephfs-secret" > parameters:<key:"csiProvisionerSecretNamespace" value:"default" > parameters:<key:"monitors" value:"10.233.7.109:6790,10.233.41.231:6790" > parameters:<key:"pool" value:"cephfs_data" > parameters:<key:"provisionVolume" value:"true" > secrets:<key:"adminID" value:"admin" > secrets:<key:"adminKey" value:"admin" > secrets:<key:"userID" value:"admin" > secrets:<key:"userKey" value:"admin" > 
`
i think we  should not log the secrets

@rootfs ^^

**Comment by rootfs** [comment]
i am using csi-v1.0 branch, secret is stripped there

```
GRPC request: {"secrets":"***stripped***","target_path":"/var/lib/kubelet/plugins/kubernetes.io/csi/volumeDevices/publish/pvc-63221849-19cc-11e9-9008-54e1ad486b31","volume_capability":{"AccessType":{"Block":{}},"access_mode":{"mode":1}},"volume_context":{"adminid":"admin","csiNodePublishSecretName":"csi-rbd-secret","csiNodePublishSecretNamespace":"default","csiProvisionerSecretName":"csi-rbd-secret","csiProvisionerSecretNamespace":"default","imageFeatures":"layering","imageFormat":"2","monitors":"rook-ceph-mon-b.rook-ceph.svc.cluster.local:6790","pool":"rbd","storage.kubernetes.io/csiProvisionerIdentity":"1547669046197-8081-csi-rbdplugin","userid":"admin"},"volume_id":"csi-rbd-vol-63255be3-19cc-11e9-a82d-54e1ad486b31"}
```

**Comment by Madhu-1** [comment]
even am using the using csi-v1.0 `quay.io/k8scsi/csi-provisioner:v1.0.0`

**Comment by Madhu-1** [comment]
Fixed in #161
